services/harmony/app/models/job.ts

```diff
@@ -409,21 +408,19 @@ export class Job extends DBRecord implements JobRecord {
   /**
    * Returns an array of all jobs that match the given constraints
    *
-   * @param transaction - the transaction to use for querying
+   * @param tx - the transaction to use for querying
    * @param constraints - field / value pairs that must be matched for a record to be returned
-   * @param getLinks - whether or not to get job links
    * @param currentPage - the index of the page to show
    * @param perPage - the number of results per page
    * @returns a list of all of the user's jobs
    */
   static async queryAll(
-    transaction: Transaction,
-    constraints: JobQuery = {},
-    getLinks = true,
+    tx: Transaction,
+    constraints: JobQuery = { where: {} },
     currentPage = 0,
     perPage = 10,
   ): Promise<{ data: Job[]; pagination: ILengthAwarePagination }> {
-    const items = await transaction('jobs')
+    const items = await tx('jobs')
```

Comment:
For consistency you might want to change this to `Job.table`

Reply:
Done